README.md

```diff
@@ -2,63 +2,207 @@
+
+## Quick Start
+
+Get started writing Terraform configurations with VS Code in three steps:
+
+- **Step 1:** If you haven't done so already, install [Terraform](https://www.terraform.io/downloads)
+
+- **Step 2:** Install the [VS Code Terraform extension](https://marketplace.visualstudio.com/items?itemName=HashiCorp.terraform).
+
+- **Step 3:** To activate the extension, open any folder or VS Code workspace containing Terraform files. Once activated, the Terraform language indicator will appear in the bottom right corner of the window.
+
+New to Terraform? Read the [Terraform Learning guides](https://learn.hashicorp.com/terraform)
+
+See [Usage](#usage) for more detailed getting started information.
+
+Read the [troubleshooting guide](#troubleshooting) for answers to common questions.
 
 ## Features
 
-- Manages installation and updates of the [Terraform Language Server (terraform-ls)](https://github.com/hashicorp/terraform-ls), exposing its features:
-  - Completion of initialized providers: resource names, data source names, attribute names
-  - Diagnostics to indicate HCL errors as you type
-  - Initialize the configuration using "Terraform: init" from the command palette
-  - Run `terraform plan` and `terraform apply` from the command palette
-  - Validation diagnostics using "Terraform: validate" from the command palette or a `validateOnSave` setting
-- Includes syntax highlighting for `.tf` and `.tfvars` files -- including all syntax changes new to Terraform 0.12
-- Closes braces and quotes
-- Includes `for_each` and `variable` syntax shortcuts (`fore`, `vare`, `varm`)
+- [IntelliSense](#autocomplete-and-intellisense) Edit your code with auto-completion of providers, resource names, data sources, attributes and more
+- [Syntax validation](#syntax-validation) Diagnostics using `terraform validate` provide inline error checking
+- [Syntax highlighting](#syntax-highlighting) Highlighting syntax from Terraform 0.12 to 1.X
+- [Code Navigation](#code-navigation) Navigate through your codebase with Go to Defintion and Symbol support
+- [Code Formating](#code-formating) Format your code with `terraform fmt` automatically
+- [Code Snippets](#code-snippets) Shortcuts for commmon snippets like fore_each and variable
+- [Terraform Module Explorer](#terraform-module-explorer) View all modules and providers referenced in the currentyl open document.
+- [Terraform commands](#terraform-commands) Directly execute commands like `terraform init` or `terraform plan` from the VS Code Command Palette.
+
+### IntelliSense and Autocomplete
+
+IntelliSense is a general term for a variety of code editing features including: code completion, parameter info, quick info, and member lists. IntelliSense features are sometimes called by other names such as autcomplete, code completion, and code hinting.
```

Comment:
What do you mean by "parameter info, quick info, and member lists" here?

Reply:
This is taken straight from the documentation: https://code.visualstudio.com/docs/editor/intellisense

When we complete attributes or provide hover documentation, that's `parameter info` and `quick info`. The general `member lists` can refer. The language fails here because there is so much variance between what language call different features. 

This was an attempt to generically describe what intellisense is, not what Terraform language constructs are so we can use the word intellisense throughout the document and know we're using the same definition as the user.